text/src/main/scala/org/apache/pekko/stream/connectors/text/javadsl/TextFlow.scala

```diff
@@ -30,10 +30,7 @@ object TextFlow {
    * Decodes a stream of bytes into a stream of characters, using the supplied charset.
    */
   def decoding(incoming: Charset): Flow[ByteString, String, NotUsed] =
-    pekko.stream.scaladsl
-      .Flow[ByteString]
-      .via(new CharsetDecodingFlow(incoming))
-      .asJava
+    pekko.stream.javadsl.Flow.fromGraph(new CharsetDecodingFlow(incoming))
```

Comment:
`pekko.stream.javadsl.Flow.fromGraph` will be use the same `scaladsl` in the end, this change may break the consistency of code style, you may want to use `pekko.stream.scaladsl.Flow.fromGraph(....).asJava`

https://github.com/apache/pekko/blob/098ab4a298bc5805b9eb851314de88fd0bef4f39/stream/src/main/scala/org/apache/pekko/stream/javadsl/Flow.scala#L78-L82

Reply:
update by using scaladsl.TextFlow